### ISSUE TITLE
SingleChildScrollView with ConstrainedBox produces unwanted divider lines

### ISSUE THREAD

**Issue by Boehrsi** [flutter/flutter#42927]
With the [given](https://api.flutter.dev/flutter/widgets/SingleChildScrollView-class.html) SingleChildScrollView example for _Expanding content to fit the viewport_ small lines are displayed between elements within the Column, mostly if the view is scrolled. It seems like the background is displayed between the elements.

I created a minimal example which is derived from the given example (only added another view and extended the height of the elements, also changed the color for easier visibility of the bug):
```dart
class Test extends StatelessWidget {
  
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: LayoutBuilder(builder: (BuildContext context, BoxConstraints viewportConstraints) {
        return LayoutBuilder(
          builder: (BuildContext context, BoxConstraints viewportConstraints) {
            return SingleChildScrollView(
              child: ConstrainedBox(
                constraints: BoxConstraints(
                  minHeight: viewportConstraints.maxHeight,
                ),
                child: IntrinsicHeight(
                  child: Column(
                    children: <Widget>[
                      Container(
                        color: Colors.deepPurple,
                        height: 200.0,
                      ),
                      Container(
                        color: Colors.deepPurple,
                        height: 200.0,
                      ),
                      Container(
                        color: Colors.deepPurple,
                        height: 200.0,
                      ),
                      Expanded(
                        child: Container(
                          color: Colors.deepPurple,
                          height: 120.0,
                        ),
                      ),
                    ],
                  ),
                ),
              ),
            );
          },
        );
      }),
    );
  }
}
```

## Steps to Reproduce

1. Implement the [SingleChildScrollView example](https://api.flutter.dev/flutter/widgets/SingleChildScrollView-class.html) for **Expanding content to fit the viewport** (or just use the given minimal example for the error)

**Target Platform:** Android
**Target OS version/browser:** Android 8
**Devices:** Samsung S7 / Pixel 2 emulator / Nexus 5 emulator

## Logs

Please have a look at the attached screenshot:

![Screenshot_20191017-132721_OX COI Messenger Dev](https://user-images.githubusercontent.com/7944013/67011449-e87f0800-f0ef-11e9-87d5-eaacb65e9d8f.jpg)

**Comment by janmoppel** [comment]
Hello @Boehrsi !

Thank you for posting an issue.
Can you please also attach the `flutter doctor -v` command output?

**Comment by sroddy** [comment]
I think it's a dupe of this: https://github.com/flutter/flutter/issues/14288

**Comment by HansMuller** [comment]
@sroddy - that's correct.

This is a known problem with antialiasing and mapping from logical pixels to device pixels. There's an open issue for a documentation fix: #14288

More info about the problem can be found here:
https://github.com/flutter/flutter/issues/14288#issuecomment-361375791
https://github.com/flutter/flutter/issues/17084#issuecomment-385718108
https://github.com/flutter/flutter/issues/15035#issuecomment-370028740

I'm going to close this as a duplicate.

**Comment by Boehrsi** [comment]
@sroddy and @HansMuller thanks for the information and pointing me to possible workarounds / solutions. And sorry for creating the dupe, could have searched a bit more I think. @janmoppel I think the Flutter doctor output isn't needed anymore, would have added it tomorrow morning.
